### ISSUE TITLE
Fullnumber in carrier number EditText

### ISSUE THREAD

**Issue by tianzhuqiao** [hbb20/CountryCodePickerProject#324]
Thanks for the great project. I know the carrier number EditText is designed for carrier number (without country code); however looks like sometime users may type in the full number (e.g. copied, or auto filled). In this case, ccp.isValidFullNumber will return false. Is there some good way to detect that and return valid full phone number?

Thanks

**Comment by hbb20** [comment]
That's interesting... It shouldn't be giving true or try form a valid number out of *possible* valid number. But I started something like, if edittext gets number pasted with "+", it might detect and load country and remove country code from it leaving valid carrier number in editText. I will look into this. Any thoughts?

**Comment by tianzhuqiao** [comment]
Right, maybe we can do something like [this](https://stackoverflow.com/questions/16809508/extract-code-country-from-phone-number-libphonenumber) (e.g., if the number is parsed successfully, then remove the country code from the number, and set the ccp accordingly if necessary; otherwise no change?).

Do you have plan to support this in future release?
